### DISCUSSION TITLE
Where can I modify the # of concurrent rsync operations?

### DISCUSSION THREAD

**Discussion by nizoo91** [ericaltendorf/plotman#853] [Q&A]
Hi,

Is there a way to allow concurrent rsync file transfers to different drives?
I realize rsync is sometimes used to transfer files over network, thus limiting it to one file at a time makes sense, however I am moving files in my local machine and it isn't fast enough to move them one at a time;

Anybody figured out how to do that? :) Thank you

**Comment by altendky** [comment]
That feature has been discussed but no implementation has been started for it yet.

**Reply by nizoo91** [reply]
Ouch, that's too bad, most of us are plotting way faster than plots being moved now with Madmax.
I guess I will have to resort back to moving plots manually using my bash mv script :/

A completely different request, another great option would be to allow multiple temp 2 flags to be specified. (-2 flag on Madmax). 

As an example:
- I am utilizing a 220G ramdisk for -2 flag currently but have 3 NVMe's
- I am usually able to run 4 plots in parallel with Madmax but not with Plotman due to the lack of ability to specify a secondary -2 (being another NVMe instead of the ramdisk)


Plotman is great none the less, just needs a bit more options to allow better plotting using Madmax! Thank you!

**Reply by altendky** [reply]
There's always more to implement, yep.  I just started using madMAx this week for my pool plots.  So, as I find time, more "madMAx oriented" features may pop up.  Though, I'm a bit skeptical of the claim that "most" people are plotting faster than a hard disk can be written to.  `:]`  This evening I'm working on generalizing plotman's support for multiple plotters.  There's sort of BladeBit and WhisperPlot is in work, so we want to have a solid base for expanding as people want to play with those.

A possible workaround for the tmp2 situation would be a larger ramdisk backed with swap on the NVMe.  I have not thought much about this myself but have seen others mention it.

Back on archiving, there is https://github.com/rjsears/chia_plot_manager as well.  I don't know exactly where the line is right now between using that and plotman archiving, or where that line should be, but I do generally expect that there is such a line.

I feel like there are more issues/discussions about this, but I'll at least cross-link to https://github.com/ericaltendorf/plotman/issues/766.

**Reply by nizoo91** [reply]
That's awesome, you will really like madMAx and hopefully add more features 👍 

I am glad you suggested the workaround with the swap partition as I was thinking the same but didn't dare to try it haha! Will give this one a go!

I will checkout the suggested plot_manager, although my script is doing fine, requires a bit of manual work to change directories when HDD is full 👎 

Thank you for your time, good luck on this great project!

**Reply by jonathanhatch** [reply]
I'd like to add a feature request: Bladebit support. That is a situation where a buffer drive and an rsync would be beautiful.

**Reply by altendky** [reply]
I have an in-work PR to clean up the handling of multiple supported plotters.  That will finish first.  Then we can work on BladeBit support.

**Reply by jonathanhatch** [reply]
Awesome. Thank you for the update. I'll be sure to donate some funds.

**Reply by altendky** [reply]
Since it was mentioned here, here's preliminary BladeBit support.  https://github.com/ericaltendorf/plotman/pull/916#issuecomment-909206239
